src/content/autoscan_inotify.h

```diff
@@ -164,7 +164,7 @@ class AutoscanInotify {
         int wd;
     };
 
-    std::unique_ptr<std::unordered_map<int, std::shared_ptr<Wd>>> watches;
+    std::unordered_map<int, std::shared_ptr<Wd>> watches;
 
```

Comment:
The argument (0) should be moved here (`std::make_unique<std::unordered_map<int, std::shared_ptr<Wd>>>(0)`)

Reply:
IIRC this was added by me to avoid an fanalyzer false positive. 0 is the default value.